docs/naivebayes.md

```diff
@@ -0,0 +1,108 @@
+Naive Bayes Classifiers
+=======================
+
+The Naive Bayes classifiers is a simple probabilistic classification model based on Bayes Theorem. It is a simple, yet powerful classification model. Since Naive Bayes classifiers simply classifies sets of data by which class has the highest likelihood, Naive Bayes classifiers can use any dsitribution or model that outputs a probability.
+
+An IPython notebook example demonstrating a Naive Bayes classifier using multivariate distributions can be [found here](https://github.com/jmschrei/pomegranate/blob/master/examples/naivebayes_multivariate_male_female.ipynb).
+
+Naive Bayes implements the following methods:
+
+```
+predict_log_proba( X ) : Input a set of data and output the log probability of each sample under each distribution
+predict_proba( X ) : Input a set of data and output the probability of each sample under each distribution
+predict( X ) : Input a set of data and classify each sample under one of the distributions
+fit( X, y ) : Fit the Naive Bayes classifier to a set of training data, with X being the sample, and y being the correct classification for each sample
+```
+
+## Initialization
+
+Naive Bayes can be initialized a number of ways. The classifier can be initialized by either passing in the initial distribution objects or by passing in the constructor and the number of components.
```

Comment:
* "Naive Bayes can be initialized either by (1) passing in a list of components (distributions or models) or (2) passing in the constructor to a distribution and the number of components."